app/models/user.rb

```diff
@@ -16,6 +16,7 @@ class User < ActiveRecord::Base
     :pto_hours_for_date_range, to: :days
 
   scope :active, -> { where(active: true) }
+  scope :time_zones, -> { pluck('DISTINCT time_zone') }
```

Comment:
Let's make this a class method rather than a scope, since it can't be chained any further. And let's use double quotes while we're at it! :)